run_route_scripts/run_with_server.py

```diff
@@ -18,9 +18,11 @@ def get_post_bodies(filename):
       line = line[0:line.rfind('}') + 1]
       post_body = json.loads(line)
       post_body['id'] = str(line_number)
+      # 1 - depart_at
+      post_body['date_time'] = {"type":1,"value":"2021-04-15T08:00"}
       yield post_body
 
-def initialize(args_):
+def initialize(args_,response_count_):
```

Comment:
the rest fo the changes in this file are already in master right? i swore you already fixed macos for this script what am i missing here?

Reply:
just wasnt merged with master